Assets/MixedRealityToolkit.SDK/Inspectors/UX/Interactable/PressableButtonInspector.cs

```diff
@@ -10,15 +10,46 @@ namespace Microsoft.MixedReality.Toolkit.Editor
     [CustomEditor(typeof(PressableButton))]
     public class PressableButtonInspector : UnityEditor.Editor
     {
-        private static bool editingEnabled = true;
+        // Struct use to store state of preview
+        // This lets us display accurate info while button is being pressed
+        private struct ButtonInfo
+        {
+            // Convenience fields for box collider info
+            public Bounds touchCageLocalBounds;
+            // Start pos for touch, also the origin of our cage
+            public Vector3 touchStartOrigin;
+            // Start and end pos for moving content
+            public float startPos;
+            public float endPos;
+            // Press, touch and release positions in z axis
+            public float pressDistPos;
+            public float touchStartPos;
+            public float releaseDistPos;
+            // Cage values
+            public float touchCageCenter;
+            public float touchCageSize;
+            // The actual values that the button uses
+            public float maxPushDistance;
+            public float pressDistance;
+            public float releaseDistanceDelta;
+        }
+
+        const string EditingEnabledKey = "MRTK_PressableButtonInspector_EditingEnabledKey";
+        const string VisiblePlanesKey = "MRTK_PressableButtonInspector_VisiblePlanesKey";
+        private static bool EditingEnabled = false;
```

Comment:
I believe private bools should start with lower case. It would be good to [review the coding guidelines here](https://microsoft.github.io/MixedRealityToolkit-Unity/CodingGuidelines.html#naming-conventions).

Reply:
This is following a convention set by other inspectors. No issue with making them lower case, but we should do it in another PR where we change the other inspectors to be consistent.